tests/README.md

```diff
@@ -63,6 +63,20 @@ IPA_SERVER_HOST=<ipaserver_host_or_ip> pytest -rs
 
 For a complete list of options check `pytest --help`.
 
+### Types of tests
+
+#### Playbook tests
+
+The playbook tests are the tests the will run our roles / modules with various parameters. Most of those tests will also run same block twice and expect ansible to not report changes ensuring idempotence. In  general those tests don't verify the state of the machine after the playbook is executed.
+
+To select only those tests use the option `-m "playbook"`
+
+#### Python tests (pytests)
+
+The pytests are tests that will execute small playbooks and then will verify the test results immediately after using python code for that.
+
+To select only those tests on a test execution use the option `-m "not playbook"`.
```

Comment:
Please, change "those" to "these".

I prefer "pytests", or "no playbook", or "python" as the parameter option for `-m`.

Reply:
The reason why it's using only playbook is because playbook tests are automatically generated so we can easily add the decorator to all tests automatically. To use either `pytest` or `python` as marker all pytest tests would need to be manually decorated with the marker and if we forget that the test would not run.

`no playbook` is not an option because the `-m` options is actually a pytest option called "marker expression". The negative  `not` is part of the expression syntax and we can't change it.

Reply:
I see. That's fine then.